src/sql/workbench/contrib/editData/browser/editDataGridPanel.ts

```diff
@@ -1151,26 +1152,21 @@ export class EditDataGridPanel extends GridParentComponent {
 		let cellClasses = 'grid-cell-value-container';
 		/* tslint:disable:no-null-keyword */
 		let valueMissing = value === undefined || value === null || (Services.DBCellValue.isDBCellValue(value) && value.isNull) || value === 'NULL';
-		let isStringNull = (Services.DBCellValue.isDBCellValue(value) && !value.isNull && value.displayValue === 'NULL');
 		if (valueMissing) {
 			valueToDisplay = 'NULL';
 			cellClasses += ' missing-value';
 		}
-		else if (isStringNull) {
```

Comment:
why are we removing the special handling of NULL strings?  I don't see anything in `getCellDisplayValue` handling this and it's not clear to me how this is related to the issue linked in the description.